chainer_chemistry/models/ggnn.py

```diff
@@ -65,22 +65,22 @@ def __call__(self, atom_array, adj):
         Returns:
             ~chainer.Variable: minibatch of fingerprint
         """
-        # reset state
-        self.update_layer.reset_state()
```

Comment:
Can you implement `reset_state` method for GGNN, and reset all the update_layers in the `reset_state` method?